server.js

```diff
@@ -0,0 +1,56 @@
+var createError = require('http-errors');
+var express = require('express');
+var path = require('path');
+var cookieParser = require('cookie-parser');
+var logger = require('morgan');
+var hbs = require('hbs');
+
+var indexRouter = require('./routes/index');
+
+
+var app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, './web/views'));
+hbs.registerHelper('math', function(lvalue, operator, rvalue) {lvalue = parseFloat(lvalue);
+        rvalue = parseFloat(rvalue);

```

Comment:
Use Prettier and editor config to format your code consistently.

Reply:
Done